beacon-chain/sync/subscriber_beacon_blocks.go

```diff
@@ -224,6 +224,8 @@ func (s *Service) processDataColumnSidecarsFromExecution(ctx context.Context, so
 			}
 
 			if len(unseenIndices) > 0 {
+				columnRecoveredFromELTotal.Add(float64(len(unseenIndices)))
```

Comment:
The help states"
> Count the number of times data columns have been recovered from the execution layer.

Here, because there is `len(unseenIndices)`, we do not count the "number of times data columns"..., but direcvtion "the number of data columns recovered".